src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -90,5 +98,18 @@ public int MaximumUnauthorizedExceptionsBeforeRestart
         /// Gets the options to control the correlation information upon the receiving of Azure Service Bus messages in the <see cref="AzureServiceBusMessagePump"/>.
         /// </summary>
         public AzureServiceBusCorrelationOptions Correlation { get; } = new AzureServiceBusCorrelationOptions();
+
+        /// <summary>
+        /// Gets the default consumer-configurable options for Azure Service Bus Queue message pumps.
+        /// </summary>
+        internal static AzureServiceBusMessagePumpOptions DefaultQueueOptions { get; } = new AzureServiceBusMessagePumpOptions
```

Comment:
This is very confusing because queues & topics are different

Reply:
I changed this so it both sets the settings in the property getter. Makes it more logical.